### ISSUE TITLE
Create Desktop Navbar

### ISSUE THREAD

**Issue by RavindraP04** [Clueless-Community/seamless-ui#895]
One need to make this component using `HTML` and `Tailwind CSS`. I would suggest to use [Tailwind Playgrounds](https://play.tailwindcss.com/) to make things faster and quicker.

Here is a reference to the component. 
![image](https://user-images.githubusercontent.com/98400348/207668985-c2fb21ff-0622-4111-9699-a817f5934109.png)

After building the component please raise a PR with a screenshot of the component and add the component in `Navbar/src/`. 
  
If you need to use any icon please use it from [Hero Icons](https://heroicons.com/)

Good luck.

**Comment by RavindraP04** [comment]
@Rajdip019 Please assign this to me

**Comment by Rajdip019** [comment]
Al, of these has done already, I guess.

**Comment by RavindraP04** [comment]
> All, of these has done already, I guess.

Yes, Instead of BrandName just SeamLess UI with logo has been used in all of them.
